e2e/src/test/java/io/aiven/kafka/tieredstorage/e2e/SingleBrokerTest.java

```diff
@@ -79,6 +80,20 @@
 import static org.assertj.core.api.Assertions.fail;
 import static org.awaitility.Awaitility.await;
 
+/**
+ * Tiered Storage end-to-end test on a Single Broker:
+ * This test includes the following stages:
+ * <ul>
+ *     <li>{@link SingleBrokerTest#remoteCopy()}: ensure that Kafka TS is moving files to remote storage</li>
+ *     <li>{@link SingleBrokerTest#remoteRead()}: ensure fetch request retrieve data from remote segments</li>
+ *     <li>{@link SingleBrokerTest#remoteManualDelete()}: delete records manually and ensure data is removed
+ *     from remote storage</li>
+ *     <li>{@link SingleBrokerTest#remoteCleanupDueToRetention()}: wait for retention by size to remove data
+ *     from remote storage</li>
+ *     <li>{@link SingleBrokerTest#topicDelete()}: (unsupported) delete topic and validate data is removed
```

Comment:
unsupported == not implemented yet?

Reply:
Correct, it should be implemented eventually: https://issues.apache.org/jira/browse/KAFKA-15166